### PR TITLE
Handling nulls in test_sensisity_ks.py sorting

### PR DESCRIPTION
**Pull request by corey-nm**
Generated by running `NM_ML_GENERATE_ONNX_TEST_DATA=1 pytest -s tests/sparseml/onnx/optim/test_sensitivity_ks.py`

### CODE REVIEW

File: tests/sparseml/onnx/optim/test_sensitivity_ks_data/mobilenet_perf.json [hunk 1]
```diff
@@ -12,79 +12,79 @@
       [
         0.0,
         [
-          0.0010810000000000001,
```
Comment (corey-nm) [inline tests/sparseml/onnx/optim/test_sensitivity_ks_data/mobilenet_perf.json:15]: the stuff in this file seems very different

File: tests/sparseml/onnx/optim/test_sensitivity_ks_data/mobilenet_perf.json [hunk 2]
```diff
@@ -1413,142 +1413,1141 @@
...
       "baseline_average": null,
-      "sparse_average": 0.4950039,
-      "sparse_integral": 7.800000000000002e-06,
-      "sparse_comparison": 0.0
+      "sparse_average": 0.3650208666666667,
+      "sparse_integral": 2.5015999999999992e-05,
+      "sparse_comparison": -5.899999999999994e-06
+    },
+    {
+      "id": null,
```
Comment (corey-nm) [inline tests/sparseml/onnx/optim/test_sensitivity_ks_data/mobilenet_perf.json:1628]: many extra layers are below this line? not sure where they came from

File: tests/sparseml/onnx/optim/test_sensitivity_ks_data/resnet50_perf.json [hunk 3]
```diff
@@ -12,79 +12,79 @@
...
-          0.006616000000000002,
-          0.006617,
-          0.006632000000000002,
-          0.006642000000000001,
-          0.006670000000000001,
-          0.006795000000000001,
-          0.006798,
-          0.006803000000000001,
-          0.006833000000000001
+          0.011909,
```
Comment (corey-nm) [inline tests/sparseml/onnx/optim/test_sensitivity_ks_data/resnet50_perf.json:15]: This file also seems very different

**Review by dbogunowicz — COMMENTED** [review]
lgtm

**Review by bfineran — APPROVED** [review]

**Review by dbogunowicz — APPROVED** [review]

**Pull request merged**